update to latest counter_culture

scbafk opened the pull request:
getting lots of deprecated notices in specs from this gem

key88sf committed:
Allow updating of specific instance to trigger Papertrail versioning.

key88sf committed:
Allow updating of specific instance to trigger Papertrail versioning.

key88sf committed:
Merge pull request #1 from diaandco/papertrail_touch

Allow updating of specific instance to trigger Papertrail versioning.

key88sf committed:
Touch papertrail version *before* counter update.

This is needed because papertrail stores object states before they change, not after.

key88sf committed:
Merge pull request #2 from diaandco/papertrail_before

Touch papertrail version *before* counter update.

key88sf committed:
Add spec for papertrail support.

magnusvk committed:
Support Rails 3

magnusvk committed:
Manage schema in schema.rb, not migrations

magnusvk committed:
Rails 5.1 now released

magnusvk committed:
Allow people to manage their own Ruby version / gemset

magnusvk committed:
Merge pull request #176 from magnusvk/pr/172

Updated Papertrail support

magnusvk committed:
Rails 5.1 now released

magnusvk committed:
Trigger Travis build

magnusvk committed:
Version bump to 1.7.0

magnusvk committed:
Update changelog

magnusvk committed:
Regenerate gemspec for version 1.7.0

magnusvk committed:
Fix typo

magnusvk committed:
Correctly quote table names and use aliases consistently

So that using this with PostgreSQL and schema namespaces will work
correctly.

bencollab committed:
Fix version check (should check ActiveRecord instead of Rails)

bencollab committed:
Use rubygems directly to determine ActiveRecord version

magnusvk committed:
Merge pull request #188 from magnusvk/quote-alias-consistently

Correctly quote table names and use aliases consistently

magnusvk committed:
Merge pull request #190 from bencollab/check-ar-version

Fix version check (should check ActiveRecord instead of Rails)

magnusvk committed:
Version bump to 1.8.0

magnusvk committed:
Update Changelog

magnusvk committed:
Regenerate gemspec for version 1.8.0

bencollab committed:
Check ActiveRecord version instead of Rails version in Reconciler

magnusvk committed:
Skip PaperTrail tests in Rails 4.0

As they currently fail. We should be able to remove this if
https://github.com/airblade/paper_trail/pull/988 gets merged.

magnusvk committed:
Update changelog

magnusvk committed:
Version bump to 1.8.1

magnusvk committed:
Regenerate gemspec for version 1.8.1

magnusvk committed:
Update README.md

mattias-lundell committed:
Fix typo - :except -> :exclude

Commit:
make use of batch_size parameter

add batch_size to whitelisted options passed to Reconciler

magnusvk committed:
Merge pull request #199 from mattias-lundell/patch-1

Fix typo - :except -> :exclude

magnusvk committed:
Merge pull request #200 from fishbrain/dont-ignore-batch-size-parameter

make use of batch_size parameter

magnusvk committed:
Update changelog

magnusvk committed:
Version bump to 1.8.2

magnusvk committed:
Regenerate gemspec for version 1.8.2

masatooba committed:
Fix README block argument is missing

magnusvk committed:
Merge pull request #202 from masatooba/fix-readme-delta-magnitude

Fix README block argument is missing

bradleesand commented:
I wanna take a look at this after the syncro launch. I need to make sure that my changes (the reason I forked this repo) still work. Maybe make some regression tests.

StephenCharles commented:
sounds good!

abaidan committed:
[fix] generate migration template with version for rails >= 5

magnusvk committed:
Change generated migration files to use new hash syntax

Because it's almost 2018. Fixes #203

magnusvk committed:
Merge branch 'master' into bugfix_migration_template_for_rails_5

magnusvk committed:
Merge branch 'master' into bugfix_migration_template_for_rails_5

magnusvk committed:
Merge pull request #205 from howtwizer/bugfix_migration_template_for_rails_5

[fix] generate migration template with version for rails >= 5

magnusvk committed:
Skip PaperTrail test in old Rails versions where it fails

magnusvk committed:
Version bump to 1.9.0

magnusvk committed:
Update changelog

magnusvk committed:
Regenerate gemspec for version 1.9.0

bakunyo committed:
Update counter_culture_migration.rb.erb

Unnecessary space in migration template.

magnusvk committed:
Merge pull request #211 from bakunyo/patch-1

Update counter_culture_migration.rb.erb

magnusvk committed:
Only mark as active when we actually change something

… to address the fact that Rails 5.1.5 changes what `saved_changes` it
reports when triggering both an `after_update` and `after_create`
callback in the same transaction.

magnusvk committed:
Specifically test Rails 5.1.4 and 5.1.5

… because that update caused a test failure and I want to make sure my
code works both before and after.

magnusvk committed:
Regenerate gemspec for version 1.9.0

magnusvk committed:
Version bump to 1.9.1

magnusvk committed:
Regenerate gemspec for version 1.9.1
